provisioning/vars.yml

```diff
@@ -0,0 +1,27 @@
+---
+# Please refer to https://github.com/idi-ops/ansible-nodejs/blob/master/defaults/main.yml
+# for documentation related to these variables
+
+nodejs_app_name: universal
+
+nodejs_app_tcp_port: 8081
+
+# Node.js version required by application (supported versions: 0.10.36*, 0.10.40, 4.1.1)
```

Comment:
node 4.2.1 is our LTS version  - but the current version of universal only supports 0.10.x